samples/kubernetes/end2end/docs/08-alert.md

```diff
@@ -0,0 +1,81 @@
+## Firing Alerts
+In this step we show you how to fire alerts.
+
+### Configuring Alerts
+We do all the alerting related configurations in the customized [values.yaml](../prometheus/values.yaml) when installing the Prometheus chart.  
+
+First, configure alerting rules. For demonstration purpse, We define one [alerting rule](../prometheus/values.yaml#L45) named 'ClusterWarning' that will fire alerts if some WLS cluster has only one running server.
+
+Second, configure Alertmanager. In [alertmanager.yml](../prometheus/values.yaml#L27) we define one route and one receiver that will send all the received alerts to a webhook.
+
+### Verifying Configuration
+Access the Prometheus alerts page in your browser with the URL `http://<HostIP>:30000/alerts`. Check the configured alert rules. You should get something like below.  
+
+![Alert Rule](./images/alert-rule.png)  
+
+Go to Alert Manager web UI via the URL `http://<HostIP>:32000`. 
+- Check the configured rule and receiver via the URL `http://<HostIP>:32000/#/status`. You should get something similar as below.  
```

Comment:
what is something?

Reply:
Same as above describe what something stands for.  For e.g. "You should get and alert configuration similar as below. "